pkg/smokescreen/smokescreen.go

```diff
@@ -421,15 +421,74 @@ func hasPort(s string) bool {
 	return strings.LastIndex(s, "]") < strings.LastIndex(s, ":")
 }
 
-// normalizeHost returns host (as string) and port (as int) derived from the
+func _normalizePort(s string) (port int, err error) {
```

Comment:
OOC why the leading underscore?

Reply:
Honestly? Because I was lazy and considered them undocumented private functions. I added some comments and removed underscores. Thanks for providing some motivation! :)